File: src/main/java/com/tolanguage/tolanguage/config/property/CacheProperty.java
package com.tolanguage.tolanguage.config.property;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.ConstructorBinding;

@ConstructorBinding
@ConfigurationProperties(prefix = "cache")
public record CacheProperty(
        Long userMaximumSize,
        Long userExpiredTimeHours
) {
}

File: src/main/kotlin/com/tolanguage/config/property/CacheProperty.kt
package com.tolanguage.config.property

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding
import java.time.Duration

@ConstructorBinding
@ConfigurationProperties(prefix = "application.cache")
data class CacheProperty(
    val userMaximumSize: Long,
    val userExpiredTime: Duration
)
